crossbar/worker/router.py

```diff
@@ -90,8 +90,7 @@
 
 from crossbar.twisted.resource import JsonResource, \
     Resource404, \
-    RedirectResource, \
```

Comment:
`crossbar.twisted.resource` doesn't import the global reactor (I got rid of the re-export which did and moved it into the only usage of it down on L1065).